docs/source/blocks/settings.md

```diff
@@ -241,6 +241,24 @@ const initialBlocks = {
 
   If the choice is limited to one item, then the setting hides itself from the `listing` block settings list.
 
+## Video block configuration
+
+This section describes how to configure video blocks.
+Although only Peertube is documented here, you can use this example for other video providers.
+
+### Peertube instances
+
+You can configure the video block to use other instances of Peertube by adding them to the `allowedPeertubeInstances` array in your project configuration object, as shown in the following example.
+
+```js
+const allowedPeertubeInstances = [
+  'freediverse.com', // About: https://freediverse.com/about/instance
+  'fair.tube', // About: https://fair.tube/about/instance
+  'tube.reseau-canope.fr', // About: https://tube.reseau-canope.fr/about/instance
+  'peertube.eus', // About: https://peertube.eus/about/instance
```

Comment:
What are the "// About" comments for?

Reply:
just to get more info about the instance. Should we remove it?